src/EditCell.tsx

```diff
@@ -41,30 +42,31 @@ export default function EditCell<R, SR>({
   }, []);
 
   const { cellClass } = column;
-  className = getCellClassname(
+  const className = getCellClassname(
     column,
     cellEditingClassname,
-    typeof cellClass === 'function' ? cellClass(row) : cellClass,
-    className
+    typeof cellClass === 'function' ? cellClass(row) : cellClass
   );
 
-  function getCellContent() {
-    if (dimensions === null) return;
-    const { scrollTop: docTop, scrollLeft: docLeft } =
-      document.scrollingElement ?? document.documentElement;
-    const { left, top } = dimensions;
-    const gridLeft = left + docLeft;
-    const gridTop = top + docTop;
+  let content;
```

Comment:
I prefer doing this over having a function that is only ever called once, unconditionally.